training-service/src/app/pages/content/content.component.css

```diff
@@ -1,6 +1,6 @@
 .col-sm img {
```

Comment:
col-sm is a bootstrap class, 
please create separate css class and add to required elements instead of
overwriting bootstrap classes